migrations/postgres/2018-12-08-191238_users_add_custom_domain/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE users ADD COLUMN custom_domain VARCHAR NOT NULL DEFAULT '';
```

Comment:
Maybe it should be unique?

Reply:
that would heavily imply, in code, that we're taking #94 seriously, in that the instance then is single-user, eh?

Reply:
Hum… yes, maybe it makes sense to have the same domain for many users.